crates/weaver_semconv_gen/src/lib.rs

```diff
@@ -0,0 +1,265 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate will generate code for markdown files.
+//! The entire crate is a rush job to catch feature parity w/ existing python tooling by
+//! poorly porting the code into RUST.  We expect to optimise and improve things over time.
+
+#![deny(
```

Comment:
These `rustc` and `clippy` directives are now inherited from the main Cargo workspace (thanks to Rust 1.78) so they are redundant I think.

Reply:
Removed